2-mandatory/4-people-i-know.js

```diff
@@ -378,13 +378,23 @@ First, I want you to find all of my friends who are 35 or older.
 */
 
 let thirtyFiveOrOlder = [];
+for (let friend of friends) {
+  if (friend.age >= 35) {
+    thirtyFiveOrOlder.push(friend.name["first"])
```

Comment:
This isn't passing the test? Can you figure out why? Looking at line 443 might help, if not we can chat on Sunday